tests/PHPSA/VariableTest.php

```diff
@@ -172,4 +172,37 @@ public function testReferenceToChange()
         static::assertSame($variable->getValue(), $parentVariable->getValue());
         static::assertSame($variable->getType(), $parentVariable->getType());
     }
+
+    public function testGetTypeName()
+    {
+        $int = new Variable('a', 1, CompiledExpression::INTEGER);
+        static::assertSame("integer", $int->getTypeName());
```

Comment:
`Static` keyword is a late static binding
Used when you need to suggest Runtime to check what class is used, mostly to call static methods from not `parent` when you extends from it class
Please change it to `parent`

Thanks 🍰